### PR TITLE
(TK-160) more valid conf parser tests

### PR DESCRIPTION
**Pull request by cprice404**
i'm sorry, and, you're welcome :)

### CODE REVIEW

**Comment by cprice404** [comment]
Added a commit that just uncomments a couple of tests that are passing now.

**Comment by KevinCorcoran** [comment]
Took a quick skim, looks good enough to me.  :ship:

**Comment by fpringvaldsen** [comment]
I'll give this a review now.

File: lib/hocon/impl/config_null.rb [hunk 1]
```diff
@@ -3,7 +3,13 @@
 require 'hocon/impl'
 require 'hocon/config_value_type'
 
-class Hocon::Impl::ConfigNull < Hocon::Impl::AbstractConfigValue
+class Hocon::Impl::ConfigNull
+  include Hocon::Impl::AbstractConfigValue
+
+  def initialize(origin)
+    super(origin)
```
Comment (fpringvaldsen) [inline lib/hocon/impl/config_null.rb]: Will this still work now that the parent is a module instead of a class?

**Comment by fpringvaldsen** [comment]
:+1: I only skimmed the 10 billion ParseTest calls, but everything looks good! I'm thinking later on we should come back and clean up some of the Javadoc specific tags, possibly even convert it to YARD or something.

File: lib/hocon/impl/config_null.rb (hunk 1 above)
Comment (fpringvaldsen) [inline lib/hocon/impl/config_null.rb]: Actually, it must work, since we're parsing configurations will `null` in them in tests and they aren't failing.

**Pull request merged**